src/map.rs

```diff
@@ -103,6 +101,8 @@ pub struct HashMap<K, V, S = crate::DefaultHashBuilder> {
     /// next element count value upon which to resize the table.
     size_ctl: AtomicIsize,
 
+    counter: ConcurrentCounter,
```

Comment:
Any particular reason why you moved where this field is in the struct?

Reply:
Oh I normally use the formatter in vscode and it probably moved it for whatever reason, sorry about that.